README.md

```diff
@@ -39,3 +39,192 @@ Please follow [Angular-cli documentation](https://github.com/angular/angular-cli
+                "collection": {},
+                "publication": "none"
+            },
+            "directurl": {
+                "caption": "Direct URL",
+                "type": "urls",
+                "protocol": [
+                    "https",
+                    "http",
+                    "ftp"
+                ],
+                "refreshSessionInterval": 180
+            },
+            "geo": {
+                "caption": "GEO",
+                "type": "urls",
+                "protocol": [
+                    "geo"
+                ],
+                "refreshSessionInterval": 180
+            }
+        }
+    }
+}
+```
+
+## macOS
+To build relocatable App on macOS with preinstalled `meteor`, `node` and `git` run the following commands. Optionally, the default programs versions used in the script can be redefined in `.env` file provided in the same folder as `build_macos.sh` script.
+
+```bash
+git clone https://github.com/datirium/scidap-satellite.git
+cd ./scidap-satellite
+npm install
+cd ./build-scripts
+./build_macos.sh
+cd ..
+npm run electron:mac
+```
+
+After script finishes running, you will find a `scidap-satellite.app` in `./release/mac` folder. All the temporary data is kept in `../build` folder and can be removed unless you want to save some time when rerunning `build_macos.sh` script next time. `Services` folder is used to save all necessary files before assembling application. This folder is cleaned on each `build_macos.sh` run.
+
+In case you by mistake run `build_ubuntu.sh` script, you will need to remove `node_modules`, `build`, `build_ubuntu`, `Services`, `ubuntu_post_build` folders in the root of you repository and rerun `npm install`, `build_macos.sh` and `npm run electron:mac` commands as it's listed above.
+
+The default configuration file `./configs/scidap_default_settings.json` will be used by Electron for generating `config.json`.
+
+_____
+## Notes for developers
+
+**Applied patches and used binaries**
+
+1. For Ubuntu bundle had to patch `cwltool/sandboxjs.py` as it fails to evaluate JavaScript expressions when CWL-Airflow is run from PM2. Perhaps, it's somehow related to how PM2 starts the project. As a workaround, made `cwltool` always run JavaScript expression evaluation using Docker.
+2. Couldn't compile aria with `--with-openssl` even if openssl was installed. As a solution use builds from here https://github.com/q3aql/aria2-static-builds
+3. PostgeSQL binaries are downloaded from https://www.enterprisedb.com/download-postgresql-binaries 
+
+**Upgrading from the old Ubuntu satellite installation**
+1. Stop services
+```bash
+systemctl stop airflow-apiserver
+systemctl stop airflow-scheduler
+systemctl stop airflow-webserver
+systemctl stop aria2c
+systemctl stop scidap-satellite
+```
+2. Create dump from the current MongoDB and stop `mongodb` service
+```
+mongodump
+systemctl stop mongodb
+```
+3. Start only `mongod` using PM2 `--only` parameter and restore database from the the dump
```

Comment:
Would not be the total command be easier? `pm2 ... --only ...` ?

Reply:
Updated